src/platform/forms-system/src/js/review/ReviewChapters.jsx

```diff
@@ -83,6 +84,7 @@ class ReviewChapters extends React.Component {
               chapterFormConfig={chapter.formConfig}
               chapterKey={chapter.name}
               form={form}
+              formConfig={formConfig}
```

Comment:
To make maintenance easier later, pass only the minimum data necessary. In this case, that looks like `formConfig.reviewErrors` instead of the whole config. (Same applies for making this component only require the `reviewErrors` as a prop itself.)

If we pass the whole `formConfig`, it's harder to determine what changes to that `formConfig` object might affect. Additionally—though it doesn't affect us in this case because `formConfig` isn't changing—passing the minimum data required is better for performance since changes to unused data won't trigger a re-render.

(To be clear, I know the forms library often doesn't follow this rule. Consider it a bad example to follow. I'd work on refactoring it if I wasn't already planning on deprecating this whole library in favor of a simpler one. :sweat_smile:)